src/sentry/runner/commands/cleanup.py

```diff
@@ -238,99 +236,33 @@ def is_filtered(model: type[Model]) -> bool:
             project, organization, days, deletes
         )
 
+        # This does not use the deletions code path, but rather uses the BulkDeleteQuery class
+        # to delete records in bulk (i.e. does not need to worry about child relations)
         run_bulk_query_deletes(
-            bulk_query_deletes,
             is_filtered,
             days,
             project,
             project_id,
             models_attempted,
         )
 
-        debug_output("Running bulk deletes in DELETES")
-        for model_tp, dtfield, order_by in deletes:
-            debug_output(f"Removing {model_tp.__name__} for days={days} project={project or '*'}")
-
-            if is_filtered(model_tp):
-                debug_output(">> Skipping %s" % model_tp.__name__)
-            else:
-                models_attempted.add(model_tp.__name__.lower())
-                imp = ".".join((model_tp.__module__, model_tp.__name__))
-
-                q = BulkDeleteQuery(
-                    model=model_tp,
-                    dtfield=dtfield,
-                    days=days,
-                    project_id=project_id,
-                    order_by=order_by,
-                )
-
-                for chunk in q.iterator(chunk_size=100):
-                    task_queue.put((imp, chunk))
-
-                task_queue.join()
-
-        project_deletion_query, to_delete_by_project = prepare_deletes_by_project(
-            project, project_id, is_filtered
+        run_bulk_deletes_in_deletes(
+            task_queue,
+            deletes,
+            is_filtered,
+            days,
+            project,
+            project_id,
+            models_attempted,
         )
 
-        if project_deletion_query is not None and len(to_delete_by_project):
```

Comment:
This code has been moved within:
https://github.com/getsentry/sentry/pull/101066/files#diff-af1cde82f065157a87af5b9c019c93a9e93aeb22661494449dbca1628e3cf306R592